### ISSUE TITLE
Peer dependency does not exist

### ISSUE THREAD

**Issue by aschbacd** [grubersjoe/react-activity-calendar#5]
When I try to install the package I get an error that `react-tooltip` is not available in version `17.0.0`.

```
npm i react-activity-calendar
...
npm ERR! Could not resolve dependency:
npm ERR! peer react-tooltip@"^17.0.0" from react-github-calendar@3.0.1
npm ERR! node_modules/react-github-calendar
npm ERR!   react-github-calendar@"*" from the root project
```

I had a look and saw the latest version is [4.2.21](https://www.npmjs.com/package/react-tooltip).
Is there a way to resolve this?

BR

**Comment by aschbacd** [comment]
I resolved it for now by using `npm i react-activity-calendar --legacy-peer-deps`.
Nevertheless the version in `package.json` does not exist.

**Comment by grubersjoe** [comment]
Hey. Thanks for pointing this out! have fixed the version number in `v1.3.3`.
